src/__tests__/ReactWrapper-spec.js

```diff
@@ -506,18 +506,20 @@ describeWithDOM('mount', () => {
         render() {
           return (
             <div>
-              {this.props.items.map(x=>x)}
+              {this.props.items.map(x => x)}
             </div>
           );
         }
       }
       matchesRender(<Foo items={['abc', 'def', 'hij']} />);
       matchesRender(
-        <Foo items={[
-          <i key={1}>abc</i>,
-          <i key={2}>def</i>,
-          <i key={3}>hij</i>,
-        ]} />
+        <Foo items=
+          {[
```

Comment:
it does, doesn't it.. but yes, it seemed the nicest way I could make the linter happy.